cityhive/app/views/inspections.py

```
@@ -0,0 +1,118 @@
+from cityhive.domain.inspection import (
+    HiveNotFoundError,
+    InspectionCreationInput,
+    InvalidScheduleError,
+)
+from cityhive.infrastructure.logging import get_logger
+from cityhive.infrastructure.typedefs import db_key, inspection_service_factory_key
+
+logger = get_logger(__name__)
+
+
+async def create_inspection(request: web.Request) -> web.Response:
```

Pylint (3.3.7)

[refactor] Too many return statements (8/6)

(R0911)